### PR TITLE
Issue/15

### PR DESCRIPTION
**Pull request by jrrbru**
Would you be willing to pull these fixes in? I'd prefer to continue to refer to your repo rather than use our fork, in case we get out of date.

I added the tests you suggested with one change: in the final test to verify that the SuperClass's association remains, I am testing both siblings just for completeness's sake.

### CODE REVIEW

File: src/backbone.associate.js [hunk 1]
```diff
@@ -153,14 +153,18 @@
...
       // Patch initialize method in prototype
       _wrapMethod(proto, _initialize, 'initialize');
+      
+      // If an _associations namespace exists in the prototype chain, get it
+      var inherited_associations = proto._associations;
 
-      // Add namespace for associations
-      proto._associations = {};
+      // Add namespace for associations, adding in any inherited associations
+      proto._associations = _.extend({}, proto._associations, inherited_associations);
```
Comment (rjz) [inline src/backbone.associate.js]: If we change L168 instead, we should be able to skip the changes at L175 and L159...L164:

    // at L168
    proto._associations = _.extend({}, proto._associations, associations)

**Comment by rjz** [comment]
Absolutely! One small cleanup, then this looks good to go.

**Comment by jrrbru** [comment]
I've pushed another cleanup commit. I think this gets at your point, @rjz. I did have to remove the `proto._associations = {};` from inside the `if` statement, because otherwise we lose the inherited `_associations`.

**Comment by rjz** [comment]
Awesome! Big thanks again for the detailed bug report and fix.

**Pull request merged**